### ISSUE TITLE
X is not a constructor

### ISSUE THREAD

**Issue by rickdio** [jshor/datebook#63]
Hi there, I get this error when trying to create a new Calendar Object.
It happens with all of the Calendars (ical, google, outlook, yahoo)

It will throw an error like this: `GoogleCalendar is not a constructor`

I'm using Angular 7.2.14
Datebook 1.3.5

**Comment by jamesjessian** [comment]
I'm getting the same issue. I can't the primary examples to work (even after fixing the missing commas).
```
_datebook.ICalendar is not a constructor
```
Datebook: 2.0.3 from NPM
Parcel: 1.12.3 

I can however generate URLs/iCal using the Calendars static methods:
```
import Calendars from 'datebook/src/utils/calendars.js';
const opts = {
  title: 'Happy Hour',
  location: 'The Bar, New York, NY',
  description: "Let's blow off some steam from our weekly deployments to enjoy a tall cold one!",
  start: '20190704T190000',
  end: '20190704T210000',
  recurrence: {
    frequency: 'WEEKLY',
    interval: 2,
  },
};
const url = Calendars.getMicrosoftCalendarUrl(opts);
console.log('url: ', url);
```
which is very cool.

**Comment by szasohel** [comment]
> Hi there, I get this error when trying to create a new Calendar Object.
> It happens with all of the Calendars (ical, google, outlook, yahoo)
> 
> It will throw an error like this: `GoogleCalendar is not a constructor`
> 
> I'm using Angular 7.2.14
> Datebook 1.3.5

same problem here. did you find any solution to this issue?

**Comment by jshor** [comment]
This library has been updated to export each calendar type individually; the import usage would now be:

```js
import { ICalendar } from 'datebook'
import { GoogleCalendar } from 'datebook'
import { YahooCalendar } from 'datebook'
import { OutlookCalendar } from 'datebook'
import { ICalendar } from 'datebook'
```

Import issues have been fixed as of v4.0.0.
